enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/ShortestPathAcceptanceTest.scala

```diff
@@ -31,7 +31,10 @@ class ShortestPathAcceptanceTest extends ExecutionEngineFunSuite with NewPlanner
   var nodeC: Node = _
   var nodeD: Node = _
 
-  override def databaseConfig = Map(GraphDatabaseSettings.forbid_shortestpath_common_nodes -> "false")
+  override def databaseConfig = Map(
+    GraphDatabaseSettings.forbid_shortestpath_common_nodes -> "false",
+    GraphDatabaseSettings.cypher_idp_solver_duration_threshold -> "10000")
```

Comment:
default for `cypher_idp_solver_duration_threshold ` is 1000. why do we run with non-default value?
if anything, wouldn't increasing it by 10x increase the duration of planning?